javascript-timers/interval.js

```diff
@@ -0,0 +1,12 @@
+function callBack() {
+  var text = document.querySelector('h1');
+  var currentNumber = text.textContent;
+
+  if (currentNumber > 1) {
+    text.textContent = currentNumber - 1;
+  } else {
+    text.textContent = '~Earth Beeeelooowww Us~';
+    clearInterval();
```

Comment:
Which interval are you clearing? https://developer.mozilla.org/en-US/docs/Web/API/clearInterval